src/test/scala/chiselTests/BlackBox.scala

```diff
@@ -222,4 +230,48 @@ class BlackBoxSpec extends ChiselFlatSpec {
       })
     }).getMessage should include("must have a port named 'io' of type Record")
   }
+
+  "BlackBoxes" should "sort the verilog output of their param map by param key" in {
+    val rand = scala.util.Random
+    rand.setSeed(0)
+
+    class Top(m: Map[String, Param]) extends Module {
+      val io = IO(new Bundle {})
+      val ibufds = Module(new ParameterizedBlackBox(m))
+      ibufds.io.I := clock
```

Comment:
I'm a little confused why FIRRTL is letting you only drive one of the inputs but not the other.